### ISSUE TITLE
SlipOscBridge should not relay package if the OSC packet is malformed

### ISSUE THREAD

**Issue by thomasfredericks** [RuudMulder/SlipOscBridge#2]
I use cvOSCcv in VCV Rack and sometimes VCV Rack crashes when an OSC message is relayed by SlipOscBdrige. I believe this is because SlipOscBridge sends the OSC packet even-thought it is malformed. Here is what it looks like in VCV Rack:

![image](https://user-images.githubusercontent.com/2685516/192367613-672c00cd-005a-434d-ba15-1852171ece22.png)

If I use Pure Data to do the relaying, VCV Rack does not crash as Pure Data blocks the badly formed packets.

![image](https://user-images.githubusercontent.com/2685516/192367563-68ce6610-8e2c-4587-b01b-2d1412dfd425.png)

SO basically, I think SlipOscBridge should not relay package if the OSC packet is malformed.

**Comment by RuudMulder** [comment]
I'm afraid this is really a bug in vcv rack and has to be solved there (or in your sending program). I have thought of implementing the package size test but decided against it. At the moment the program can be used for any serial-to-udp message, not just osc messages. So it would limit its possible use.

**Comment by thomasfredericks** [comment]
Hi! The issue was resolved in the cvOSCcv package for VCV Rack.
But maybe an _option/ui toggle_ could  be added to optionally check that the packets are a multiple of 4? This simple error checking could also be helpful for generic UDP.
